docs/SnapshotTesting.md

```diff
@@ -121,6 +121,20 @@ You can try out this functionality by cloning the
 [snapshot example](https://github.com/facebook/jest/tree/master/examples/snapshot),
 modifying the `Link` component, and running Jest.
 
+#### Interactive Updates
+
+Snapshots can also be updated one at a time in the Jest watch mode:
```

Comment:
This is not entirely true, as interaction with snapshots happens test file by test file, not test by test. Can you fix the description to be more accurate (including the paragraph below)?
We'd like to avoid issues about this feature not working as expected (but we'll get there eventually)

Reply:
@thymikee good catch, this is my bad as it's in my copy above

Related to this, the skip option still says "Press s to skip the current snapshot"

![](http://dp.hanlon.io/3o3G383V2m00/[1b91dca29ef4c0d5229fc87bbee16b01]_Image%202018-01-13%20at%203.45.16%20PM.png)